### DISCUSSION TITLE
Is 2.0.5 not ready for Arm64 (Raspi 4)?

### DISCUSSION THREAD

**Discussion by kenkos68** [espressif/arduino-esp32#7291] [Q&A]
2.0.4 Works fine in Arduino on Ubuntu 22.04 for Raspberry Pi 4.  When updagrading to 2.0.5, it fails on the esptool_py download.

In the package_esp32_index.json it is still trying to get esptool-3.3 which fails due to not being there and also the checksum would be wrong.
            {
              "host": "aarch64-linux-gnu",
              "url": "https://github.com/espressif/arduino-esp32/releases/download/2.0.4/esptool-3.3-linux.tar.gz",
              "archiveFileName": "esptool-4.2.1-linux.tar.gz",
              "checksum": "SHA-256:5a45fb77eb6574554ec2f45230d0b350f26f9c24ab3b6c13c4031ebdf72a34ab",
              "size": "90123"
            }

Not sure if a bug or intentional.

**Comment by me-no-dev** [comment]
Wow... seems that we have missed to update the file location. I have edited the JSONs directly. Could you try again?

**Reply by kenkos68** [reply]
Thank you!  Installs fine now!

**Comment by Jason2866** [comment]
Speaking about updates. gcc toolchains patch5 are out and the do support now Apple silicon native (no more with Rosetta).

**Reply by me-no-dev** [reply]
yeah but Arduino IDE does not look for arm64 packages yet :(

**Comment by Jason2866** [comment]
Good that Platformio support is there too. It does works nice with ;-)

**Comment by Jason2866** [comment]
@kenkos68  Give platformio a try, since IDF 4.4.3 is not yet official released it is not supported official too.
BUT we use already for Project Tasmota with our fork/build. You just have to use in Platformio this
```
platform = https://github.com/tasmota/platform-espressif32/releases/download/v2.0.5.1/platform-espressif32-2.0.5.1.zip
```

**Reply by bhupiister** [reply]
@Jason2866 i tested the new release on M5 core2 esp32 and had spiffs mount error. I guess the release is 10-15 days old, is there an update? 

I am using your compiled version as it supports putting task stack in psram.

**Reply by Jason2866** [reply]
I have a M5 stack core2 too. No problems with LittleFS. I do not use SPIFFS. Idk if there is a bug. If than it is in Arduino Core. Nothing changed here since weeks.

**Reply by bhupiister** [reply]
I reverted back to the below version and thankfully everything is working normal now.
`platform = https://github.com/tasmota/platform-espressif32/releases/download/v2.0.2idf/platform-espressif32-2.0.2.zip
`
In version **platform-espressif32-2.0.5.1.zip** I was getting **Invalid Credentials** from ssl_client.cpp when using with Firebase library

**Comment by Jason2866** [comment]
Firebase lib seems needs to be updated for actual core version.
